lib/commands/waitForExist.js

```diff
@@ -65,7 +65,9 @@ module.exports = function waitForExist(selector, ms, reverse) {
 
                 var now = new Date().getTime();
                 if(now - start > ms) {
-                    return cb(null, false);
```

Comment:
yes, sorry, that makes sense, thanks!
But it should still be:

```js
if(now - start > ms) {
    return cb(null, false);
}
```

I assume that people expect a boolean value to assert against not an error.